python-bareos/debian/control

```diff
@@ -1,13 +1,20 @@
 Source: python-bareos
-Maintainer: Joerg Steffens <joerg.steffens@bareos.com>
+Maintainer: Bareos Team <packager@bareos.com>
```

Comment:
```suggestion
Maintainer: Bareos GmbH & Co. KG <packager@bareos.com>
```

Reply:
Bareos Team is used throughout similar settings, in SPEC files, DEB, ...